Refresh.GameServer/Types/Assets/AssetDependencyRelation.cs

```diff
@@ -0,0 +1,9 @@
+using Realms;
+
+namespace Refresh.GameServer.Types.Assets;
+
+public partial class AssetDependencyRelation : IRealmObject
+{
+    public string Dependent { get; set; }
```

Comment:
should disable nullable in model classes unless necessary to avoid these compiler warnings